### ISSUE TITLE
train/main.py questions

### ISSUE THREAD

**Issue by y5131241997** [frederic-bousefsaf/ippg-3dcnn#6]
Hello, in order to train the model ,do I need to change the paths of model_conv3D.json and weights_conv3D.h5 to ../../models/weights_conv3D.h5 and ../../models/model_conv3D.json?   
For example :    232 lines   f1 = open('../../statistics_loss_acc.txt', 'a') 
this line need to change to f1 = open('../../models/statistics_loss_acc.txt', 'a') ?
I would like to ask for training models directly, or need to change other codes? 
Please advise me !

**Comment by frederic-bousefsaf** [comment]
Hello,
You should of course adapt the paths to your project
Sincerely

**Comment by y5131241997** [comment]
Ok,thank you !
So,can i train the model directly with your current version code? or do i need to add other configuration files(model_conv3D.json and weights_conv3D.h5 )by myself?

**Comment by frederic-bousefsaf** [comment]
Yes you can train the model on your data, we do not provide the weights (.h5) of the model presented in this work.
Sincerely

**Comment by y5131241997** [comment]
ok,thank you!
